examples/stm32/xively_demo_stm32f4xx_nucleo_wifi/Src/demo_io.c

```diff
@@ -342,7 +362,7 @@ int8_t io_read_humidity( float* read_value )
  * @param  dec_prec the decimal precision to be used
  * @retval None
  */
-void floatToInt( float in, int32_t* out_int, int32_t* out_dec, int32_t dec_prec )
+static void floatToInt( float in, int32_t* out_int, int32_t* out_dec, int32_t dec_prec )
```

Comment:
Good point. I lifted that function from somewhere in the MEMS SDK, name and all. I'll rename it :)